pulseguardian/web.py

```diff
@@ -129,6 +132,17 @@ def decorated_function(*args, **kwargs):
     return decorated_function
 
 
+def requires_admin(f):
+    """Decorator for views that are allowed for admin users only"""
+    @wraps(f)
+    def decorated_function(*args, **kwargs):
+        if not g.user.admin:
+            """404: Non admin user not need know that this route exists."""
```

Comment:
The wording is a bit odd on this.  How about something like: 

    404: Non-admin user does not have access to this route.